neurom/core/_soma.py

```diff
@@ -206,21 +211,27 @@ def __str__(self):
                 (repr(self._points), self.center, self.radius))
 
 
+# classes of somas
+SOMA_CONTOUR = 'contour'
+SOMA_CYLINDER = 'cylinder'
+
+
 def _get_type(points, soma_class):
     '''get the type of the soma
 
     Args:
         points: Soma points
         soma_class(str): one of 'contour' or 'cylinder' to specify the type
     '''
-    assert soma_class in ('contour', 'cylinder', )
+    assert soma_class in (SOMA_CONTOUR, SOMA_CYLINDER)
+
     npoints = len(points)
-    if soma_class == 'contour':
+    if soma_class is SOMA_CONTOUR:
```

Comment:
I would say it could cause problems and I don't see why the values of these constants could be changed.
Up to you though.